azkaban-common/src/test/java/azkaban/executor/ExecutorHealthCheckerTest.java

```diff
@@ -145,4 +146,46 @@ public void checkExecutorHealthConsecutiveFailures() throws Exception {
         + "executor-port: 12345";
     verify((this.alerterHolder).get("email")).alertOnError(eq(flow1), eq(expectedReason));
   }
+
+  /**
+   * Test that the wrapper routine swallows any exceptions reported by underlying health checker.
+   */
+  @Test
+  public void testCheckExecutorHealthWrapperExceptionHandling() throws Exception {
+    this.activeFlows.put(EXECUTION_ID_11, new Pair<>(
+        new ExecutionReference(EXECUTION_ID_11, this.executor1), this.flow1));
+    when(this.apiGateway.callWithExecutionId(this.executor1.getHost(), this.executor1.getPort(),
+        ConnectorParams.PING_ACTION, null, null)).thenThrow(new Error("test error"));
+
+    // this will throw causing the test to fail in case the error is not caught correctly
```

Comment:
A comma is missing between throw and causing.

Reply:
Will do.